scout/parse/matchmaker.py

```diff
@@ -51,7 +52,14 @@ def omim_terms(case_obj):
     case_disorders = case_obj.get("diagnosis_phenotypes")  # array of OMIM terms
     if case_disorders:
         for disorder in case_disorders:
-            disorder_obj = {"id": ":".join(["MIM", str(disorder)])}
+            omim_term = store.disease_term(disorder)
+            if omim_term is None:
+                LOG.warning(f"Disease term {disorder} could not be found in database.")
+                continue
+            disorder_obj = {
+                "id": ":".join(["MIM", str(disorder)]),
+                "label": omim_term.get("description"),
```

Comment:
> > If you remove the case from MME and re-add it, you should see that also the OMIM diagnosis will have label

> >Hm, any reason not to do that dynamically instad? It works fine like this, but it you could just check from the store when rendering?

This is current bug. In main branch we are  sending disorders described by "id" only, while according to the [MME API](https://onlinelibrary.wiley.com/doi/10.1002/humu.22850) you can also specify a label in "disorders":

<img width="837" alt="image" src="https://user-images.githubusercontent.com/28093618/153445107-fd3ac6bf-d7f7-4dae-a175-4237560e763c.png">

Reply:
Well, not really a bug since the label is not mandatory, but better to save it since we have the info, and it's also good for showing it in that web page